Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -121,9 +138,10 @@ public class Klaviyo: NSObject  {
      
      - Parameter userInfo: NSDictionary containing the push notification text & metadata
      */
+    @objc
     public func handlePush(userInfo: NSDictionary) {
-        if let body = userInfo["body"] as? NSDictionary, let _ = body["_k"] {
-            trackEvent(eventName: KLPersonOpenedPush, properties: userInfo)
+        Klaviyo.sdkInstance.handle(remoteNotification: userInfo as! [AnyHashable : Any]) { result in
```

Comment:
Is it possible to avoid forced type cast here?

Reply:
Makes sense!

Reply:
Pushing a fix now.